apps/console/src/features/identity-verification-providers/pages/identity-verification-providers.tsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { ListLayout, PageLayout, PrimaryButton } from "@wso2is/react-components";
+import React, { FunctionComponent, MouseEvent, SyntheticEvent, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { DropdownItemProps, DropdownProps, Icon, PaginationProps } from "semantic-ui-react";
+import { AppConstants, UIConstants, history } from "../../core";
+import { useIdentityVerificationProviderList } from "../api";
+import { useIDVPTemplateTypeMetadataList } from "../api/ui-metadata";
+import { IdentityVerificationProviderList } from "../components";
+import { IdentityVerificationProviderConstants } from "../constants";
+import { handleIDVPTemplateRequestError, handleIdvpListFetchRequestError } from "../utils";
+
+
+type IDVPPropsInterface = IdentifiableComponentInterface;
+
+const IDENTITY_VERIFICATION_PROVIDER_LIST_SORTING_OPTIONS: DropdownItemProps[] = [
+    {
+        key: 1,
+        text: "Name",
```

Comment:
We may need to i18n these strings :)

Reply:
No further action is needed because this section was removed.